test/modules/load-gen/Settings.cs

```diff
@@ -56,9 +60,16 @@ static Settings Create()
                 ? null
                 : configuration.GetValue<string>("testResultCoordinatorUrl");
 
-            string priorities = string.IsNullOrWhiteSpace(configuration.GetValue<string>("priorities"))
+            List<int> priorities = string.IsNullOrWhiteSpace(configuration.GetValue<string>("priorities"))
                 ? null
-                : configuration.GetValue<string>("priorities");
+                : configuration.GetValue<string>("priorities").Split(';').Select(int.Parse).ToList();
+
+            List<int> ttls = string.IsNullOrWhiteSpace(configuration.GetValue<string>("ttls"))
+                ? null
+                : configuration.GetValue<string>("ttls").Split(';').Select(int.Parse).ToList();
+
+            int ttlThresholdValue = configuration.GetValue<int>("ttlThresholdSecs");
```

Comment:
will this return default(int) or throw exception if this key is not defined?

Reply:
Default. Now will throw exception if TTL and priorities both exist and TtlThreshold is <= 0